util/qmk_install.sh

```diff
@@ -10,7 +10,7 @@ case $(uname) in
 	Linux)
 		exec "${util_dir}/linux_install.sh"
 	;;
-	MSYS_NT*)
+	MINGW64_NT*)
```

Comment:
I think that `M*_NT*` here would be better.  That way, it gets both options. 

Namely, I'd rather see that it gets both, so that we properly parse both possibilities.

Reply:
Yeah you're right, I'll get on it